db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/FromBaseTable.java

```diff
@@ -123,6 +124,12 @@
     long defaultRowCount;
     double defaultSelectivityFactor = -1d;
 
+    private boolean disableIndexPrefixIteration = false;
+
+    // Statistics about the first column in the conglomerate currently
+    // being considered as the access path of this table:
+    private FirstColumnOfIndexStats currentIndexFirstColumnStats = new FirstColumnOfIndexStats();
```

Comment:
Since the code later test on `currentIndexFirstColumnStats == null`, maybe we don't need to default initialize it to an object here?

Reply:
We need it because other code that reads currentIndexFirstColumnStats does not check for null, for example, line 3805 of FromBaseTable.  The goal is to always have it be non-null.